static/js/https/domains.js

```diff
@@ -1,7 +1,70 @@
-$(document).ready(function () {
+$(function () {
```

Comment:
I won't annotate the whole file, but the JavaScript for rendering the domains table here has been almost completely refactored.